src/ui/map.js

```diff
@@ -170,6 +172,8 @@ const defaultOptions = {
  *   bearing will snap to north. For example, with a `bearingSnap` of 7, if the user rotates
  *   the map within 7 degrees of north, the map will automatically snap to exact north.
  * @param {boolean} [options.pitchWithRotate=true] If `false`, the map's pitch (tilt) control with "drag to rotate" interaction will be disabled.
+ * @param {number} [options.dragThreshold=0] The threshold, measured in pixels, that determines how far a drag must be before it is considered a drag
```

Comment:
I have two related suggestions here:

1) I think `clickTolerance` might be a more accurate name for this option. For one thing, that's what this option is called in Leaflet (even though it's part of the `Draggable` class) which might help people who are migrating to GL-JS or searching for a solution to this problem. Additionally, the original issue for this PR was how to get a click event instead of a drag event when the shift was only a couple pixels, so really this is about setting a tolerance limit around the click event to prevent it turning into a drag.

2) I think the description could be clearer. Consider how Leaflet describes `clickTolerance`: "The max number of pixels a user can shift the mouse pointer during a click for it to be considered a valid click (as opposed to a mouse drag)." You may want to make it clear in your description that the value of the option is the max number *inclusive* since you're checking for `<=` while Leaflet just checks `<`.